pkgs/top-level/all-packages.nix

```diff
@@ -20889,7 +20889,7 @@ with pkgs;
 
   tinyxml-2 = callPackage ../development/libraries/tinyxml-2 { };
 
-  tiscamera = callPackage ../os-specific/linux/tiscamera { stdenv = gcc10StdenvCompat; };
+  tiscamera = libsForQt5.callPackage ../os-specific/linux/tiscamera { stdenv = gcc10StdenvCompat; };
```

Comment:
imho just pass `libsForQt5` in and get `wrapQtAppsHook` from there. Semantically, it's clearer b/c qt is a small part of this package.